src/main/java/blob/common/Messages.java

```diff
@@ -0,0 +1,36 @@
+package blob.common;
+
+/**
+ * Class containing the reusable messages to print to user
+ */
+public class Messages {
```

Comment:
I like how messages are all stored in a single Messages class. I think this is a very creative idea!